aws/signer/internal/v4/headers.go

```diff
@@ -53,6 +53,7 @@ var RequiredSignedHeaders = Rules{
 			"X-Amz-Storage-Class":                                         struct{}{},
 			"X-Amz-Website-Redirect-Location":                             struct{}{},
 			"X-Amz-Content-Sha256":                                        struct{}{},
+			"X-Amz-Tagging":                                               struct{}{},
```

Comment:
1. Is this a valid change for any other rest based service apart from `s3` which use this Header?

Reply:
Yeah this would be valid for all other services, as not hoisting headers is valid approach. In addition S3 is the only service that sends this header for a presigned url.